### ISSUE TITLE
GG Max - EXP Build - BUG - Move object and find surface does not work in latest EXP build

### ISSUE THREAD

**Issue by Tum1370** [TheGameCreators/GameGuruRepo#4266]
HI,
As the title says, This selection in the editor in the latest EXP build appears to not be working correctly.
I have tried placing several items and they all seem to float above the ground and i have to manually move them to the ground.

![Untitled](https://user-images.githubusercontent.com/66874403/226191444-7d161ea6-c6cf-4c7b-b49c-44cd26eea640.png)

**Comment by ZakJudges** [comment]
Hi @Tum1370, I haven't been able to reproduce this one. Could you provide a step-by-step that I can follow along to try and reproduce? Thanks :)

**Comment by Tum1370** [comment]
@ZakJudges I just tested this on the same RPG test map that i created, and placed the same 3 plants down in the same area, where they wouldnt place properly before.
This time they all placed perfect, So i am not sure why.

I will close this issue now as i cannot reproduce it myself on the same map that i had the problem with.

Thanks.

**Comment by Tum1370** [comment]
@ZakJudges  I have just found this bug again, where an item does not place on the ground.
How to produce the bug is you place one item down, say on top of a table.
Then SHIFT and CLICK the item to copy it, The copy does not place correctly.

**Comment by plemsoft** [comment]
@Tum1370 Looks like it works fine in the lastest EXP build , could you try again and let me know if its also fixed for you , Thanks for your help :)
https://www.youtube.com/watch?v=q0HGMIXdF44

**Comment by Tum1370** [comment]
@plemsoft Yes this looks like its been fixed thanks. I cannot reproduce this now. Ill close this issue now.
Thanks
